src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -2322,12 +2402,24 @@ dynamic Local(dynamic x)
         {
             return x;
         }
```

Comment:
Add `[CompilerTrait(CompilerFeature.Dynamic)]` (as aappropriate here and elsewhere)